silkworm/core/types/block.cpp

```diff
@@ -241,6 +241,7 @@ namespace rlp {
         Header rlp_head{.list = true};
         rlp_head.payload_length += length(b.transactions);
         rlp_head.payload_length += length(b.ommers);
+        rlp_head.payload_length += length(b.consensus_parameter_index.has_value());
```

Comment:
Is length(bool) consist across platforms? Maybe we should use a safer way to write this line

Reply:
yes, they have a specialized `length` for type bool https://github.com/eosnetworkfoundation/silkworm/blob/3273082db7c763b4d1051ae82f60891c236ced13/silkworm/core/rlp/encode.hpp#L70